doc/man7/EVP_MAC_HMAC.pod

```diff
@@ -0,0 +1,69 @@
+=pod
+
+=head1 NAME
+
+EVP_MAC_HMAC - The HMAC EVP_MAC implementation
+
+=head1 DESCRIPTION
+
+Support for computing HMAC MACs through the B<EVP_MAC> API.
+
+=head2 Numeric identity
+
+B<EVP_MAC_HMAC> is the numeric identity for this implementation, and
+can be used in functions like EVP_MAC_CTX_new_id() and
+EVP_get_macbynid().
+
+=head2 Supported controls
+
+The supported controls are:
+
+=over 4
+
+=item B<EVP_MAC_CTRL_SET_KEY>
+
+EVP_MAC_ctrl_str() takes to type string for this control:
+
+=over 4
+
+=item "key"
+
+The value string is used as is.
+
+=item "hexkey"
+
+The value string is expected to be a hexadecimal number, which will be
+decoded before passing on as control value.
+
+=back
+
+=item B<EVP_MAC_CTRL_SET_FLAGS>
+
+Sets HMAC flags.  This is passed directly to HMAC_CTX_set_flags().
+
+There are no corresponding string control types.
+
+=item B<EVP_MAC_CTRL_SET_ENGINE>
+
+=item B<EVP_MAC_CTRL_SET_MD>
+
+These work as described in L<EVP_MAC(3)/CONTROLS>.
+
+There are no corresponding string control types.
```

Comment:
So hmmm, the CMAC implementation does have a "cipher" string, so this should have a corresponding "digest" string, to be consistent.